src/main/java/com/atlassian/db/replica/api/reason/Reason.java

```diff
@@ -32,6 +32,8 @@ private Reason(final String name, boolean isRunOnMain, boolean isWrite) {
             new ReasonBuilder("HIGH_TRANSACTION_ISOLATION_LEVEL").isRunOnMain(true).isWrite(false).build();
     public static final Reason RO_API_CALL =
             new ReasonBuilder("RO_API_CALL").isRunOnMain(false).isWrite(false).build();
+    public static final Reason REPLICA_GET_FAILURE =
```

Comment:
Why `REPLICA_GET_FAILURE` not `REPLICA_FAILURE`

Reply:
I think it better describes the reason. We failed to get a replica connection. Maybe replica failed, maybe the connection pool is saturated.

Reply:
`REPLICA_GET_CONNECTION_FAILURE` 

I didn't known it is about connection, I thought it's about replica _any_ failure